app/js/settings.js

```diff
@@ -7,13 +7,8 @@
 var Settings = {
   BASE_SERVER: 'https://support.allizom.org',
   LOCALES: [
-    ['de', 'Deutsch'],
     ['en-US', 'English'],
-    ['fr', 'Français'],
     ['es', 'Español'],
-    ['it', 'Italiano'],
     ['pt-BR', 'Português (do Brasil)'],
```

Comment:
Bah. I thought we didn't need to worry about this in Buddyup because it only runs in ES5-using browser (aka, Firefox). But I should probably stick with the prevaling style. I'll remove it.